dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/HibernateEventChangeLogStore.java

```diff
@@ -92,6 +92,7 @@ public Page<EventChangeLog> getEventChangeLogs(
                  u.uid
           from EventChangeLog ecl
           join ecl.event e
+          left join ecl.dataElement d
```

Comment:
Because of the many to one mapping to data elements, Hibernate automatically assumes it needs an inner join, so I need to explicitly add the left join in the query